packages/@lwc/engine-core/src/framework/main.ts

```diff
@@ -51,47 +51,3 @@ export type {
     WireAdapterConstructor,
     WireAdapterSchemaValue,
 } from './wiring';
-
-// Initialization APIs for the renderer, to be used by engine implementations ----------------------
```

Comment:
removing all these public apis... with this change, core package is useless by itself because it doesn't export the renderer methods needed by a compiled template, basically the compiled code must import or be linked to a production of a package that does have an export called `renderer`.